source/includes/reference/_users.md

```diff
@@ -62,7 +62,67 @@ details.address.city        | Address city name          | Text
 details.address.state  | Address state code State code. Required if country is US, CA, AU, BR. | Text
 details.address.occupation  | User occupation. Required for US, CA, JP         | Text
 
+## Get the currently logged in user
 
+> Example Request:
+
+```shell
+curl -X GET https://api.sandbox.transferwise.tech/v1/me \
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+{
+  "id": 294205,
+  "name": "Oliver Wilson",
+  "email": "oliver@gmail.com",
+  "active": true,
+  "details": {
+    "firstName": "Oliver",
+    "lastName": "Wilson",
+    "phoneNumber": "+3725064992",
+    "occupation": "",
+    "address": {
+      "city": "Tallinn",
+      "countryCode": "EE",
+      "postCode": "12112",
+      "state": "",
+      "firstLine": "Salu tee 134"
+    },
+    "dateOfBirth": "1977-07-01",
+    "avatar": "https://lh6.googleusercontent.com/-XzeFZ2PJE1A/AAAAAAAAAAI/AAAAAAAAAAA/RvuvhXFsqs0/photo.jpg",
+    "primaryAddress": 236532
```

Comment:
Makes no sense in the response, what would we use the `primaryAddress` field for?

Reply:
You use it to look up in the address endpoints